compliance_checker/suite.py

```diff
@@ -584,6 +584,10 @@ def standard_output(self, ds, limit, check_name, groups):
         print("\n")
         print("-" * width)
         print("{:^{width}}".format("IOOS Compliance Checker Report", width=width))
+        print("{:^{width}}".format("Version {}".format(__version__), width=width))
+        print("{:^{width}}".format("Report generated {}".format(
+            datetime.now(timezone.utc).strftime("%Y-%m-%dT%H:%M:%SZ")
```

Comment:
Came here to [paste this](https://blog.ganssle.io/articles/2019/11/utcnow.html) but I'm glad you probably already know about it!